qdax/core/emitters/multi_emitter.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional, Tuple
+
+import jax
+from jax import numpy as jnp
+
+from qdax.core.containers.repertoire import Repertoire
+from qdax.core.emitters.emitter import Emitter, EmitterState
+from qdax.types import Descriptor, ExtraScores, Fitness, Genotype, RNGKey
+
+
+class MultiEmitterState(EmitterState):
+    emitters_state_tuple: Tuple[EmitterState, ...]
```

Comment:
I think we should use PyTree properties rather than Tuples, don't you think?

Reply:
I am not sure to follow what you mean. Technically, I think that Tuples are PyTrees.

How do you think this should be reorganised?

Reply:
Sorry, I did not see your answer. I was thinking batching the EmitterState. So the leafs will be batched, instead of having a tuple.

Reply:
Hey. I changed my mind about this particular point :slightly_smiling_face: